hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -796,6 +796,18 @@ will be -1. To disable readaheads, set this value to 0. If your workload is
  doing only random reads (non-sequential) or you are seeing throttling, you
   may try setting this value to 0.
 
+To run under limited memory situations configure the following.
```

Comment:
make clear: or when doing many writes in same process (bulk uploads, hive LLAP/spark with many workers)